Microsoft.Toolkit.Uwp.UI.Controls/ColumnResizer/ColumnResizer.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// ColumnResizer is a UI control that add the resizing functionality to a Grid Column.
+    /// </summary>
+    [TemplatePart(Name = RESIZERNAME, Type = typeof(Grid))]
+    public partial class ColumnResizer : Control
+    {
+        private const string RESIZERNAME = "Resizer";
+        private const int FixedOffset = 5;
+        private const double EnteredOpacity = .7;
+        private const double DefaultOpacity = 1;
+
+        private readonly CoreCursor _resizeCursor = new CoreCursor(CoreCursorType.SizeWestEast, 1);
+        private readonly CoreCursor _arrowCursor = new CoreCursor(CoreCursorType.Arrow, 1);
+        private Grid _resizer;
+        private bool _isResizing;
+
+        /// <summary>
+        /// Gets Column Resizer Container Grid
+        /// </summary>
+        private Grid Resizable => _resizer.FindVisualAscendant<Grid>();
```

Comment:
Can't you just use `this` instead of _resizer ?
Would this.Parent always be a grid? If not, this control wouldn't work, since it's not in the grid it is supposed to resize.